### DISCUSSION TITLE
Holiday mode action

### DISCUSSION THREAD

**Discussion by Sjaaker** [OlivierZal/com.melcloud#544] [Ideas]
Hi Olivier, Thanks for the great app. I am relatively new to Homey and have an Ecodan heat pump but it would be great to have an action that enables/disables the holiday mode when I am away from home (e.g., similar as through the MELCloud app and the button on the heat pump). Would that be possible to realize? Thanks!

**Comment by OlivierZal** [comment]
Hi @Sjaaker,

Thanks for your feedback :-)

I think that technically it's possible (when I read your message 2 days ago, I even made a small change to ease such a thing), but I still wonder which user experience would be the best – with the technical constraint that heat pump values and holiday mode values doesn't come from the same API.

By the way: I have no air-to-water device (only air-to-air), do you know if holiday mode is just an on/off according to a given range of dates, or if there are other actions performed by the heat pump behind the scenes?

I don't know if you've already noticed, but it's possible to enable holiday mode from the MELCloud app settings in Homey – there's a bug right now, but will be fixed soon.

**Comment by OlivierZal** [comment]
Closing in absence of answer.
